src/sw.ts

```diff
@@ -177,6 +220,37 @@ function isSwAssetRequest (event: FetchEvent): boolean {
   return isActualSwAsset
 }
 
+/**
+ * Set the expires header with a timestamp base
+ */
+function setExpiryHeader (response: Response, ttlSeconds: number = 3600): Response {
```

Comment:
`setExpiryHeader` modifies respToCache but also returns a `Response`. We should change response type to `void` since we're not using the response